hack/README.md

```diff
@@ -0,0 +1,58 @@
+docker/hack
+============================
+
+## About
+
+A collection of useful scripts that we use; we decided it deserves a great
+README! If there are any issues regarding the intention of a particular script
```

Comment:
amicable language... personally I am not such a huge fan of this. anyways, it is part of `hack`, so maybe it is in-place.